doc/classes/InputEventGesturePan.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="InputEventGesturePan" inherits="InputEventGesture" version="4.0">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
```

Comment:
Would be good to fill those, it's not immediately obvious what a pan gesture is (especially to non native speakers or people not experienced with touch events yet).